### DISCUSSION TITLE
Translated strings with any failing checks is maybe empty

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [WeblateOrg/weblate#7486] [Q&A] [answered]
I don't know if bug or misconfiguration but the filter [Translated strings with any failing checks](https://hosted.weblate.org/translate/koreader/koreader/bg/?q=has:check%20AND%20state:%3E=translated) of a project I'm contributing is empty even though there is a number before it:

![](https://user-images.githubusercontent.com/10962709/162634254-25412de0-1d7b-4d2c-8f31-3d82398ec03b.png)

Where should I report? At KOReader or Weblate?

**Comment by nijel** [comment, marked as answer]
It's not empty, it contains 9 strings for me...

**Reply by StoyanDimitrov** [reply]
It was random browser extension breaking the request.
